provision.sh

```diff
@@ -256,92 +263,142 @@ function provision {
     local stackAddr="${2}"
     local gatewayID="${3}"
     local apiKey="${4}"
+    local gatewayModel="${5:-"${wirnetStationModel}"}"
```

Comment:
IMO this may be confusing for some, @johanstokking can we break the API here and require a 5th argument(model)?